transactions/idTableStaking/delegation/delegator_add_capability.cdc

```diff
@@ -0,0 +1,22 @@
+import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+import FlowToken from 0xFLOWTOKENADDRESS
+
+// This transaction adds a public delegator capability to an account with
+// an existing NodeDelegator object
+
+transaction {
+
+    prepare(acct: AuthAccount) {
+
+        if acct.borrow<&FlowIDTableStaking.NodeDelegator>(from: FlowIDTableStaking.DelegatorStoragePath) == nil ||
```

Comment:
I've seen this "does storage contain something at this path with this type?" check a couple times before. 

Could you please open a feature request issue in the Cadence repo to add a "check" function, just like we have it for capabilities? 

This should be easy to implement and would be more idiomatic.

Thank you!

Reply:
Yeah I'll open an issue